x-pack/plugins/security_solution/public/hosts/components/kpi_hosts/risky_hosts/translations.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const HOSTS_COUNT = (quantity: number) =>
+  i18n.translate('xpack.securitySolution.kpiHosts.riskyHosts.hostsCount', {
+    defaultMessage: '{quantity} hosts',
```

Comment:
Host might be singular here. Could try doing this: https://github.com/elastic/kibana/blob/16af3e95cb00d338a3fcdae45f69101110af736e/x-pack/plugins/security_solution/public/detections/pages/detection_engine/translations.ts#L144